examples/run_CPMG.py

```diff
@@ -31,91 +31,71 @@
 # {{{set phase cycling
 phase_cycling = True
 if phase_cycling:
-    ph1_cyc = r_[0, 1, 2, 3]
-    nPhaseSteps = 4
+    Nph_overall = r_[0,1,2,3]
```

Comment:
call these `ph_overall` and `ph_diff` (`N` doesn't make sense -- these are phase lists -- when we were talking about `N` it was when it was the length of the list -- like `nPhaseSteps`)

Reply:
okay so to avoid making an extra line (e.g.`nph_overall = len(ph_overall)`) I am just calling` len(ph_overall) `where necessary (eg. setting the axis